src/main/java/com/example/bookstore/service/impl/UsersServiceImpl.java

```diff
@@ -18,8 +19,10 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.Optional;
+import java.util.List;
 
 @Service
+@Transactional
```

Comment:
```suggestion
@Transactional(readOnly=true)
```
ri nha, vào bên trong, các method nào cần readOnly flase thì em tự update lại tại method đó thôi nhá

Reply:
Da vang

Reply:
vâng mà chưa fix nè